bazel/linux/providers.bzl

```diff
@@ -77,3 +77,12 @@ RuntimeBundleInfo = provider(
         "check": "list of RuntimeInfo, executable (and its runfiles) to run OUTSIDE the VM AFTER the RUN to check if the run was successful",
     },
 )
+
+BootRamImageInfo = provider(
+    doc = """Maintains the information necessary to represent a BootRam image, needed for aarch64 qemu acf-machine boot.""",
+    fields = {
+        "name": "Name of the rule that defined this bootram image.",
```

Comment:
my gut tells me we don't need the `name` field in the provider.

if you scroll up and look at other providers in this file, none of them have a `name` field.

Reply:
Ok.  Removed it now.